### PR TITLE
metrics: 1.0 update

### PR DESCRIPTION
**Pull request by dmpetrov**
DVC 1.0 metrics

### CODE REVIEW

File: content/docs/command-reference/metrics/diff.md [hunk 1]
```diff
@@ -7,10 +7,10 @@ commits in the <abbr>DVC repository</abbr>, or between a commit and the
...
 
 ```usage
-usage: dvc metrics diff [-h] [-q | -v]
-                        [--targets [<path> [<path> ...]]]
-                        [-t <type>] [-x <path>] [-R] [--all]
-                        [--show-json] [--show-md] [a_ref] [b_ref]
+usage: dvc metrics diff [-h] [-q | -v] [--targets [<paths> [<paths> ...]]]
+                        [-R] [--all] [--show-json] [--show-md] [--no-path]
+                        [--old]
+                        [a_rev] [b_rev]
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/diff.md:13]: I'd combine these 2 lines

File: content/docs/command-reference/metrics/show.md [hunk 2]
```diff
@@ -6,8 +6,8 @@ formatting.
 ## Synopsis
 
 ```usage
-usage: dvc metrics show [-h] [-q | -v] [-t <type>] [-x <path>]
-                        [-a] [-T] [-R] [--all-commits]
+usage: dvc metrics show [-h] [-q | -v] [-a] [-T] [--all-commits] [-R]
+                        [--show-json]
                         [targets [targets ...]]
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/show.md:11]: Same. I'd combine these lines.

File: content/docs/command-reference/metrics/index.md [hunk 3]
```diff
@@ -27,21 +21,76 @@ positional arguments:
 In order to track metrics associated to machine learning experiments, DVC has
 the ability to mark a certain stage <abbr>outputs</abbr> as files containing
 metrics to track (see the `--metrics` option of `dvc run`). Metrics are
-project-specific floating-point values e.g. `AUC`, `ROC`, etc.
-
-Supported file formats: JSON. Metrics can be organized in a tree hierarchy in a
-JSON file. DVC addresses the metrics by the tree path.
+project-specific floating-point or integer values e.g. `AUC`, `ROC`,
+`False-Positives` etc.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/index.md:25]: Probably no need for back quotes here. Maybe
```suggestion
project-specific floating-point or integer values e.g. AUC, ROC,
_false positives_, etc.
```

File: content/docs/command-reference/metrics/index.md [hunk 4]
```diff
@@ -27,21 +21,76 @@ positional arguments:
...
+
+[Show](/doc/command-reference/metrics/show), and
+[diff](/doc/command-reference/metrics/diff), commands are available to manage
+<abbr>DVC project</abbr> metrics.
+
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/index.md:30]: Maybe remove these lines since they're already listed in the top definition.

File: content/docs/command-reference/metrics/index.md [hunk 5]
```diff
@@ -27,21 +21,76 @@ positional arguments:
...
+## Formats and metrics types
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/index.md:31]:
```suggestion
### Formats and types of metrics
```
(H3 if we agree this is part of the description so no need for extra entry in right sidebar)

File: content/docs/command-reference/metrics/index.md [hunk 6]
```diff
@@ -27,21 +21,76 @@ positional arguments:
...
+
+Supported file formats: JSON and YAML. Metrics can be organized in a tree
+hierarchy in a JSON file or a YAML file. DVC addresses the metrics by the tree
+path. In the example below five metrics are presented: `train.accuracy`,
+`train.loss`, `train.TN`, `train.FP` and `time_real`.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/index.md:36]: I always wanted to improve this paragrpah:
```suggestion
Metrics can be organized in a tree hierarchy in a JSON or YAML file. DVC
addresses specific metrics by the tree path. In the JSON example below,
five metrics are presented: `train.accuracy`, `train.loss`, `train.TN`, `train.FP`
and `time_real`.
```

File: content/docs/command-reference/metrics/index.md [hunk 7]
```diff
@@ -27,21 +21,76 @@ positional arguments:
...
 these numbers are produced by the model training or model evaluation code and
 serve as a way to compare and pick the best performing experiment.
 
-[Add](/doc/command-reference/metrics/add),
-[show](/doc/command-reference/metrics/show),
-[diff](/doc/command-reference/metrics/diff),
-[modify](/doc/command-reference/metrics/modify), and
-[remove](/doc/command-reference/metrics/remove) commands are available to set up
-and manage <abbr>DVC project</abbr> metrics.
+## Default metrics files
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/index.md:54]:
```suggestion
### Default metrics files
```

File: content/docs/command-reference/metrics/index.md [hunk 8]
```diff
@@ -27,21 +21,76 @@ positional arguments:
...
+
+`dvc metrics show` and `dvc metrics diff` commands by default use all metrics
+files that are specified in `dvc.yaml`. So, there no need to specify metrics
+file name to see the deault metrics. The metrics files can be added to
+`dvc.yaml` by options `--metrics` (`-m`) or `--metrics-no-cache` (`-M`) of
+`dvc run` command or manualy to `metrics` section of a stage:
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/index.md:60]:
```suggestion
`dvc metrics` subcommands use all metric files that are specified in
`dvc.yaml` by default. There's no need to specify metric file names to see
these metrics. Metric files can be added to `dvc.yaml` with the `--metrics`
(`-m`) or `--metrics-no-cache` (`-M`) options of `dvc run`, or manually to
the `metrics` section of a stage in `dvc.yaml`:
```

File: content/docs/command-reference/metrics/index.md [hunk 9]
```diff
@@ -27,21 +21,76 @@ positional arguments:
...
+    outs:
+      - model.pkl
+    metrics:
+      - summary.json:
+          cache: false
+```
+
+`False` value of the additional `cache` option specifies that this is not data
+file, it should not be moved to DVC cache and it should be commited to Git
+history instead.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/index.md:81]:
```suggestion
`cache: false` above specifies that `summary.json is not a data file: it will not
be <abbr>cached</abbr> by DVC. Metric files are normally committed with
Git instead.
```

File: content/docs/command-reference/metrics/index.md [hunk 10]
```diff
@@ -27,21 +21,76 @@ positional arguments:
...
+
+### Difference between continuous and scalar metrics
+
+DVC has two concepts for metrics for representing result of machine learning
+training or data processing:
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/index.md:86]:
```suggestion
DVC has two concepts for metrics, that represent different results of
machine learning training or data processing:
```

File: content/docs/command-reference/metrics/index.md [hunk 11]
```diff
@@ -27,21 +21,76 @@ positional arguments:
...
+
+1. `dvc metrics` to represent scalar numbers such as AUC, true positive rate and
+   others.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/index.md:89]:
```suggestion
1. `dvc metrics` represent **scalar numbers** such as AUC, _true positive
rate_, etc.
```

File: content/docs/command-reference/metrics/index.md [hunk 12]
```diff
@@ -27,21 +21,76 @@ positional arguments:
...
+2. `dvc plots` to visualize continuous metrics such as AUC curves, loss
+   functions, confusion matrices, and others.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/index.md:91]:
```suggestion
2. `dvc plots` can be used to visualize **continuous metrics** such as AUC curves,
loss functions, confusion matrices, etc.
```

File: content/docs/command-reference/metrics/index.md [hunk 13]
```diff
@@ -27,21 +21,76 @@ positional arguments:
...
+
+In this section, only scalar metrics are covered.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/index.md:93]: Probably not needed. I think it's obvious 🙂

File: content/docs/command-reference/metrics/index.md [hunk 14]
```diff
@@ -54,7 +103,7 @@ and manage <abbr>DVC project</abbr> metrics.
 
 ## Examples
 
-First, let's create a simple DVC-file:
+First, let's create a simple DVC pipeline stage:
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/index.md:106]:
```suggestion
First, let's create a simple stage:
```
Maybe link "stage" to the run cmd ref, if no other links to that doc exist nearby.

File: content/docs/command-reference/metrics/index.md [hunk 15]
```diff
@@ -80,30 +127,24 @@ $ dvc metrics show -a
...
-filter data out of metric files:
-
-```dvc
-$ dvc metrics show --xpath AUC data/eval.json
-      data/eval.json: {'AUC': 0.65115}
-```
-
-The `xpath` filter can be saved as the default way to display a metrics file:
+When metrics file changes in the user workspace witout commiting it to the Git,
+the `dvc metrics diff` command show the difference between metrics values:
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/index.md:131]: It's "metric file(s)" right? The term we want, I mean. Singular
```suggestion
When there are metric file changes (before committing them with Git), the
`dvc metrics diff` command shows the difference between metrics values:
```

Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/index.md:131]: But maybe I can further simplify this paragraph to be less specific...

File: content/docs/command-reference/metrics/index.md [hunk 16]
```diff
@@ -80,30 +127,24 @@ $ dvc metrics show -a
...
+$ dvc metrics diff
+Path       Metric    Value    Change
+eval.json  ACU       0.66729  0.01614
+eval.json  error     0.16982  0.00322
+eval.json  TP        516      -12
 ```
 
-And finally let's remove `data/eval.json` from the project metrics:
+Metrics commited to the Git history can be shown by referencing the commit name
+or any aliase:
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/index.md:142]:
```suggestion
Metric files committed with Git can be shown by referencing the commit,
branch, or tag (or any other reference):
```
Link reference to the corresponding Git doc (revision I think? Need to check the other docs where we link to git-scm.com).

Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/index.md:142]: Or maybe skip the last part about refs.

File: content/docs/command-reference/metrics/add.md [hunk 17]
```diff
@@ -1,115 +0,0 @@
-# metrics add
```
Comment (skshetry) [inline content/docs/command-reference/metrics/add.md:1]: Are we deprecating `metrics add`?

File: content/docs/command-reference/metrics/index.md (hunk 6 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/index.md:36]: Also applying this and other matching text updates to current dvc plot cmd ref...

File: content/docs/command-reference/metrics/add.md (hunk 17 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/add.md:1]: Seems so. Cc @dmpetrov

File: content/docs/command-reference/metrics/show.md [hunk 18]
```diff
@@ -98,10 +58,60 @@ compares them with a previous version.
...
+The basic use case shows the values in the current workspace:
+
+```dvc
+$ dvc metrics show
+      data/eval.json:
+		{
+		    AUC: 0.66729,
+		    error: 0.16982,
+		    TP: 516
+		}
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/show.md:70]: Shoot guys actually these examples still seem outdated i.e. not reflecting 1.0 scalar metrics. This here is a JSON array. Same throughout all the examples here and in `metrics diff`. Still we can merge this PR but we should try to address this immediately after.

File: content/docs/command-reference/metrics/add.md (hunk 17 above)
Comment (dmpetrov) [reply content/docs/command-reference/metrics/add.md:1]: yes.

File: content/docs/command-reference/metrics/show.md (hunk 18 above)
Comment (dmpetrov) [reply content/docs/command-reference/metrics/show.md:70]: right! could you please adjust the format to:
```
$ dvc metrics show
      data/eval.json:
		    AUC: 0.66729
		    error: 0.16982
		    TP: 516
```

Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/show.md:70]: OK will do. I thought the whole example was wrong because a single metrics file seems to contain several scalar values but maybe I'm just confused. Updating format...

File: content/docs/command-reference/metrics/diff.md [hunk 19]
```diff
@@ -45,28 +46,19 @@ They're calculated between two commits (hash, branch, tag, or any
...
-  DVC-file. See `dvc metrics show` for more details.
-
 - `--all` - list all metrics, even those without changes.
 
 - `--show-json` - prints the command's output in easily parsable JSON format,
   instead of a human-readable table.
 
 - `--show-md` - prints the command's output in Markdown table format.
 
+- `--old` - Show old metric value in addition to the new value.
```
Comment (shcheklein) [inline content/docs/command-reference/metrics/diff.md:56]: Show -> show

(and in a few other places)

File: content/docs/command-reference/metrics/index.md [hunk 20]
```diff
@@ -1,47 +1,104 @@
...
-[diff](/doc/command-reference/metrics/diff),
-[modify](/doc/command-reference/metrics/modify), and
-[remove](/doc/command-reference/metrics/remove) commands are available to set up
-and manage <abbr>DVC project</abbr> metrics.
+DVC has two concepts for metrics, that represent different results of machine
+learning training or data processing:
+
+1. `dvc metrics` represent **scalar numbers** such as AUC, _true positive rate_,
+   etc.
+2. `dvc plots` can be used to visualize **continuous metrics** such as AUC
```
Comment (shcheklein) [inline content/docs/command-reference/metrics/index.md:36]: continuous - @dmpetrov are we okay using the term continuous after all, even though it's not very mathematically correct?

File: content/docs/command-reference/remote/list.md [hunk 21]
```diff
@@ -1,12 +1,6 @@
 # remote list
 
-List all available data remotes.
```
Comment (shcheklein) [inline content/docs/command-reference/remote/list.md:3]: @jorgeorpinel no related to this PR? Please let's try to do this in Regular updates or separate PRs

File: content/docs/command-reference/plots/show.md [hunk 22]
```diff
@@ -1,6 +1,7 @@
 # plots show
 
-Generate a plot image from from a [metrics](/doc/command-reference/plots) file.
+Generate a plot image from from a
+[continuous metrics](/doc/command-reference/plots) file.
```
Comment (shcheklein) [inline content/docs/command-reference/plots/show.md:4]: may be we can use "series"? (like in one of the descriptions above?)

File: content/docs/command-reference/metrics/index.md [hunk 23]
```diff
@@ -1,47 +1,104 @@
...
 
 In order to track metrics associated to machine learning experiments, DVC has
 the ability to mark a certain stage <abbr>outputs</abbr> as files containing
-metrics to track (see the `--metrics` option of `dvc run`). Metrics are
-project-specific floating-point values e.g. `AUC`, `ROC`, etc.
+metrics to track.
 
-Supported file formats: JSON. Metrics can be organized in a tree hierarchy in a
-JSON file. DVC addresses the metrics by the tree path.
+Metrics are project-specific floating-point or integer values e.g. AUC, ROC,
```
Comment (shcheklein) [inline content/docs/command-reference/metrics/index.md:25]: I think it's better for types of metrics go before Description in this case (and in plots)

Comment (shcheklein) [reply content/docs/command-reference/metrics/index.md:25]: at least the part that covers "we have two types ...."

File: content/docs/command-reference/metrics/index.md [hunk 24]
```diff
@@ -1,47 +1,104 @@
...
+_false positives_, etc. They can be defined with the `-m` (`--metrics`) and `-M`
+options of `dvc run`.
```
Comment (shcheklein) [inline content/docs/command-reference/metrics/index.md:27]: include (--metrics-no-cache) or remote `--metrics`

File: content/docs/command-reference/remote/list.md (hunk 21 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/remote/list.md:3]: Yeah sorry I decided to just standardize how all the commands with indices link to the subcommands the same way its done in metrics. I've been sloppy with PR separation lately because of all the battlefronts but will be more careful.

File: content/docs/command-reference/metrics/index.md [hunk 25]
```diff
@@ -1,47 +1,104 @@
...
+  },
+  "time_real": 344.61309599876404
+}
+```
+
+DVC itself does not ascribe any specific meaning for these numbers. Usually they
+are produced by the model training or model evaluation code and serve as a way
+to compare and pick the best performing experiment.
+
+### Default metrics files
```
Comment (shcheklein) [inline content/docs/command-reference/metrics/index.md:74]: metric files (metrics files sounds really bad from english lang perspective)

File: content/docs/command-reference/plots/show.md (hunk 22 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/plots/show.md:4]: I also vote for series and/or data series here.

File: content/docs/command-reference/metrics/index.md (hunk 25 above)
Comment (shcheklein) [reply content/docs/command-reference/metrics/index.md:74]: I would this into Description - it just a regular basic overview and it overlaps a lot with some parts in Description

Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/index.md:74]: I thought I got all of these before, seems like this one is missing. Yes. We have either "metric file", "metrics file" (which I'm not so sure of and may need reviewing, or "metric files" for plural.

File: content/docs/command-reference/metrics/show.md [hunk 26]
```diff
@@ -6,9 +6,8 @@ formatting.
 ## Synopsis
 
 ```usage
-usage: dvc metrics show [-h] [-q | -v] [-t <type>] [-x <path>]
-                        [-a] [-T] [-R] [--all-commits]
-                        [targets [targets ...]]
+usage: dvc metrics show [-h] [-q | -v] [-a] [-T] [--all-commits] [-R]
+                        [--show-json] [targets [targets ...]]
```
Comment (shcheklein) [inline content/docs/command-reference/metrics/show.md:10]: need to add `--show-json` to the list of options

File: content/docs/command-reference/metrics/diff.md [hunk 27]
```diff
@@ -7,10 +7,9 @@ commits in the <abbr>DVC repository</abbr>, or between a commit and the
 ## Synopsis
 
 ```usage
-usage: dvc metrics diff [-h] [-q | -v]
-                        [--targets [<path> [<path> ...]]]
-                        [-t <type>] [-x <path>] [-R] [--all]
-                        [--show-json] [--show-md] [a_ref] [b_ref]
+usage: dvc metrics diff [-h] [-q | -v] [--targets [<paths> [<paths> ...]]]
+                        [-R] [--all] [--show-json] [--show-md] [--no-path]
```
Comment (shcheklein) [inline content/docs/command-reference/metrics/diff.md:11]: same here - make sure we have all options described at least on some basic level

File: content/docs/command-reference/metrics/index.md [hunk 28]
```diff
@@ -1,47 +1,104 @@
...
-[remove](/doc/command-reference/metrics/remove).
+A set of commands to manage, collect, and display _project metrics_:
+[show](/doc/command-reference/metrics/show), and
+[diff](/doc/command-reference/metrics/diff).
 
 ## Synopsis
 
 ```usage
 usage: dvc metrics [-h] [-q | -v]
-                   {show,add,modify,remove,diff} ...
```
Comment (shcheklein) [inline content/docs/command-reference/metrics/index.md:14]: remove them from the highlighter

File: content/docs/command-reference/metrics/diff.md [hunk 29]
```diff
@@ -80,7 +72,7 @@ Start by creating a simple metrics file and commit it:
 
 ```dvc
 $ dvc run -M metrics.json \
-        'echo {\"AUC\": 0.9643, \"TP\": 527} > metrics.json'
+          'echo {\"AUC\": 0.9643, \"TP\": 527} > metrics.json'
```
Comment (kurianbenoy) [inline content/docs/command-reference/metrics/diff.md:75]: On running this command in dvc version 1.0.0a6:

I get the following error:
error: `-n | --name'

File: content/docs/command-reference/metrics/show.md [hunk 30]
```diff
@@ -98,10 +58,60 @@ compares them with a previous version.
 
 ## Examples
```
Comment (kurianbenoy) [inline content/docs/command-reference/metrics/show.md:59]: In the examples, wouldn't it be better to include an example with metrics file in format of CSV  or TSV's rather than always using JSON files

File: content/docs/command-reference/metrics/diff.md (hunk 19 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/diff.md:56]: Fixing. Didn't find any other instances though.

File: content/docs/command-reference/metrics/index.md (hunk 20 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/index.md:36]: FYI: The term "continuous metrics" is currently used 10 times.

File: content/docs/command-reference/metrics/index.md (hunk 24 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/index.md:27]: Added ``(`--metrics-no-cache`)`` but Idk what you meant by "remote `--metrics`"

File: content/docs/command-reference/metrics/index.md (hunk 25 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/index.md]: Added this to #608

File: content/docs/command-reference/metrics/diff.md (hunk 27 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/diff.md:11]: They are 🙂

File: content/docs/command-reference/metrics/index.md (hunk 28 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/index.md:14]: Already gone 🙂

Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/index.md:14]: It's config\prismjs\dvc-commands.js right?

File: content/docs/command-reference/metrics/index.md (hunk 23 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/index.md]: I tried doing this and it didn't make much sense but I'm making some changes so Types comes earlier in plots and also use the term "scalar metrics" in metrics so it's obvious there's a distinction. See ee87e99f (sorry, also has a bunch of small updates to plots refs — related to this PR)

File: content/docs/command-reference/metrics/index.md (hunk 28 above)
Comment (shcheklein) [reply content/docs/command-reference/metrics/index.md:14]: yep, name sounds right

File: content/docs/command-reference/metrics/diff.md (hunk 29 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/diff.md]: The examples weren't ready yet. Please try the latest version and lmk.

File: content/docs/command-reference/metrics/show.md (hunk 30 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/show.md:62]: Those formats are no longer supported by `dvc metrics`.

File: content/docs/command-reference/metrics/diff.md [hunk 31]
```diff
@@ -45,28 +46,19 @@ They're calculated between two commits (hash, branch, tag, or any
...
 - `--all` - list all metrics, even those without changes.
 
 - `--show-json` - prints the command's output in easily parsable JSON format,
   instead of a human-readable table.
 
 - `--show-md` - prints the command's output in Markdown table format.
 
+- `--old` - show old metric value in addition to the new value.
+
+- `--no-path` - Don't show metric path in the result table. This option is
```
Comment (shcheklein) [inline content/docs/command-reference/metrics/diff.md:58]: Don't -> don't

File: content/docs/command-reference/metrics/index.md [hunk 32]
```diff
@@ -1,47 +1,104 @@
...
-Supported file formats: JSON. Metrics can be organized in a tree hierarchy in a
-JSON file. DVC addresses the metrics by the tree path.
+These metrics are project-specific floating-point or integer values e.g. AUC,
+ROC, _false positives_, etc. They can be defined with the `-m` (`--metrics`) and
+`-M` (`--metrics-no-cache`) options of `dvc run`.
 
-DVC itself does not ascribe any specific meaning for these numbers. Usually
-these numbers are produced by the model training or model evaluation code and
-serve as a way to compare and pick the best performing experiment.
+### Types of metrics
```
Comment (shcheklein) [inline content/docs/command-reference/metrics/index.md:29]: this should go before description

File: content/docs/command-reference/metrics/index.md [hunk 33]
```diff
@@ -1,47 +1,104 @@
...
+  },
+  "time_real": 344.61309599876404
+}
+```
+
+DVC itself does not ascribe any specific meaning for these numbers. Usually they
+are produced by the model training or model evaluation code and serve as a way
+to compare and pick the best performing experiment.
+
+### Default metric files
```
Comment (shcheklein) [inline content/docs/command-reference/metrics/index.md:74]: should become part of the description

File: content/docs/command-reference/metrics/show.md [hunk 34]
```diff
@@ -17,61 +16,22 @@ positional arguments:
...
-[DVC-files](/doc/user-guide/dvc-file-format). If `targets` are provided, it will
-show those specific metric files instead. With the `-a` or`-T` options, this
-command shows the different metrics values across all Git branches or tags,
-respectively.
+[DVC-files](/doc/user-guide/dvc-file-format).
 
-The optional `targets` argument can contain one or more metric files. With the
-`-R` option, some of the target can even be directories, so that DVC recursively
-shows all metric files inside.
+> Metrics can be defined with the `-m` (`--metrics`) and `-M` options of
```
Comment (shcheklein) [inline content/docs/command-reference/metrics/show.md:21]: either include --metric-no-cache or remove `--metrics`

File: content/docs/command-reference/plots/index.md [hunk 35]
```diff
@@ -20,62 +20,45 @@ positional arguments:
 ## Description
 
 DVC provides a set of commands to visualize _continuous metrics_ of machine
-learning experiments. Usual examples of plots are AUC curves, loss functions,
-and confusion matrices.
+learning experiments (see []()). Usual examples of plots are AUC curves, loss
```
Comment (shcheklein) [inline content/docs/command-reference/plots/index.md:23]: `[]()`?

File: content/docs/command-reference/plots/index.md [hunk 36]
```diff
@@ -20,62 +20,45 @@ positional arguments:
...
+`dvc plots` subcommands can work with metric files committed to a Git repo
+history, data files controlled by DVC, or any other file in file system.
 
 DVC generates plots as HTML files that can be open with a web browser. These
 HTML files use [Vega-Lite](https://vega.github.io/vega-lite/). Vega is a
 declarative grammar for defining plots using JSON. The plots can also be saved
 as SVG or PNG image filed from the browser.
 
-### Difference between continuous and scalar metrics
+### Types of metrics
```
Comment (shcheklein) [inline content/docs/command-reference/plots/index.md:38]: same here - should go before description

File: content/docs/command-reference/metrics/index.md (hunk 33 above)
Comment (shcheklein) [reply content/docs/command-reference/metrics/index.md:74]: do we need a similar paragraph in plots?

File: content/docs/command-reference/metrics/diff.md (hunk 31 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/diff.md:58]: OK checked all these in da12a1a and 86b8f62e (the latter for a separate PR).

File: content/docs/command-reference/plots/index.md (hunk 35 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/plots/index.md:23]: Oops... ⏳

File: content/docs/command-reference/plots/index.md (hunk 36 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/plots/index.md:38]: Will reply in https://github.com/iterative/dvc.org/pull/1360#pullrequestreview-421446357.

File: content/docs/command-reference/metrics/index.md (hunk 32 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/index.md]: I didn't think you meant that literally earlier since it breaks our standard cmd ref structure. But OK, did it now in 3b898b37 (included also continuous and scalar terminology update discussed in Slack).

WDYT? I'm not so sure about this one...

File: content/docs/command-reference/metrics/index.md (hunk 33 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/index.md:73]: It is part of the description.

Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/index.md:73]: > do we need a similar paragraph in plots?

No, plots require specifying a target/datafile

Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/index.md:73]: p.s. this led me to discover the following issues: #3925 and #3924

Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/index.md:73]: I mean iterative/dvc/issues/3925 and iterative/dvc/issues/3924

File: content/docs/command-reference/metrics/index.md (hunk 32 above)
Comment (shcheklein) [reply content/docs/command-reference/metrics/index.md]: > standard cmd ref structure.

we should not be following or trying to preserve it a cost of actual meaning and natural flow of the text. I don't know if this should go before or should be part of description, but introducing types and explaining the difference should be before explaining a particular type.

File: content/docs/command-reference/metrics/index.md (hunk 24 above)
Comment (shcheklein) [reply content/docs/command-reference/metrics/index.md]: remote == remove

File: content/docs/command-reference/metrics/index.md (hunk 33 above)
Comment (shcheklein) [reply content/docs/command-reference/metrics/index.md:72]: it overlaps a lot:

For example:

> Metric files can be added to `dvc.yaml` with the `--metrics` (`-m`) or
`--metrics-no-cache` (`-M`) options of `dvc run`, or manually to the `metrics`
section of a stage in `dvc.yaml`:

and

> This kind of metrics can be defined with the `-m` (`--metrics`) and `-M`
(`--metrics-no-cache`) options of `dvc run`.

Why can't we make it a single paragraph?

anyway, probably not a big deal .. we can keep it like this for now

File: content/docs/command-reference/metrics/index.md (hunk 32 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/index.md]: Would be tricky putting it in the Description before the basic explanation unless we remove the **Types of metrics** header and just make it into the first paragraph.

But for now I think it's fine actually, since this change only affects the index.md of both metrics and plots, which is kind of a placeholder cmd ref anyway. Other indices are used to explain concepts more than as cmd refs also, like for cache and remote 🙂 so we can be more flexible about these in terms of doc structure going fwd.

File: content/docs/command-reference/metrics/index.md (hunk 33 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/index.md:72]: > > do we need a similar paragraph in plots?
>
> No, plots require specifying a target/datafile

Actually probably we'll need it but this will be done as a result of resolving iterative/dvc/issues/3925 (I'll keep track).

> it overlaps a lot:

Yeah I didn't notice before. Removed redundancy in 47ad3116 (part of #1382)!

> Why can't we make it a single paragraph?

I could but would probably want to completely remove the dvc.yaml example in that case and the `cache:false` note. Also not sure it makes sense to have this info before the **Supported file formats** section. No strong opinion, I'll do it if you think it's best. Please lmk
